e2e-tests/cypress/integration/customizer/form-fields/form-fields.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe('Form fields', function () {
+	beforeEach('Sets up customizer', function () {
```

Comment:
This could be a before hook since it doesn't change any data between each request inside the describe block.